### DISCUSSION TITLE
API v10?

### DISCUSSION THREAD

**Discussion by ghost** [jagrosh/MusicBot#1119] [Questions] [answered]
Has MusicBot been updated for the newest API version? it doesn't take them too long to completely shutdown the older APIs most of the time.

**Comment by jagrosh** [comment, marked as answer]
It hasn't, but it will be before the version its on is shut down.

Also, it's demonstrably false that it "doesn't take them long to shut down older APIs"... Version 6 has been the default - while also being deprecated - for roughly 2 years.
